app/javascript/controllers/auto_citation_controller.js

```diff
@@ -60,14 +60,18 @@ export default class extends Controller {
         const surname = this.contributorLasts[index].value
         return `${surname}, ${firstInitial}`
       }
-      return this.contributorOrgTargets[index].value
+      return this.contributorOrgNames[index].value
     })
   }
 
   get contributorRoles() {
     return this.contributorRoleTargets.filter(elem => elem.disabled == false)
   }
 
+  get contributorOrgNames() {
+    return this.contributorOrgTargets.filter(elem => elem.disabled == false)
+  }
+
```

Comment:
this ensures we only find organization names that are not disabled when building the citation...this mimics what we do for names 

i think we should have had this change before this work too, maybe it still worked with JS errors :shrug: but this normalizes it